docs/Spatial.md

```diff
@@ -1,6 +1,93 @@
 #Phidget Spatial Sensor
-More commonly known as an accelerometer.
+More commonly known as an accelerometer, the PhidgetSpatial library makes for intuitive and lightning fast development without any compromise. All spacial directions must be defined by user, for an in dept guide into this, [see your board's user guide](http://www.phidgets.com/products.php?category=1). For a quick start into your Spatial project see this [basic Manager example](https://github.com/RIAEvangelist/node-phidget-API/blob/master/examples/spatial.js).
 
-[Basic spatial example](https://github.com/RIAEvangelist/node-phidget-API/blob/master/examples/spatial.js)
+##Methods
 
-ToDo : map out module and describe use
\ No newline at end of file
+|Method call|Parameters|Description|
+|---|---|---|
+|[__phidget__.connect](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md)|[__phidget__.params object](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md#connecting--phidgetparams)|This connects the Phidget Spatial|
+|[__phidget__.whenReady](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md)|function |This executes a function when the Phidgets RFID is ready to be used.|
+|[__phidget__.quit](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md)|N/A |This method requests a disconnect from the phidget board.  The disconnected event will be dispatched when the connection has been successfully disconnected. |
+|observeGyro|'gyro'|used for asynchronously observing the changes to the PhidgetSpatial board's gyroscopic sensors. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework.|
+|observeAcceleration|'accel'|used for asynchronously observing the changes to the PhidgetSpatial board's accelerometers. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework. |
```

Comment:
the whole board is more commonly known as an accelerometer so this would be confusing. I would say accelleration axis or something else maybe.